src/ui/public/vis_maps/kibana_map.js

```diff
@@ -0,0 +1,507 @@
+import { EventEmitter } from 'events';
```

Comment:
On tilemaps you're currently able to zoom out all the way, and then there are multiple earths but only one of them has the data on it:

![multiple-earths](https://cloud.githubusercontent.com/assets/627123/24294600/99467d82-106d-11e7-8692-d16212cfdb6f.gif)